pkg/sql/distsqlrun/processors.proto

```diff
@@ -548,6 +548,8 @@ message JobProgress {
 // See ccs/sqlccl/csv.go for implementation.
 message ReadCSVSpec {
   optional roachpb.CSVOptions options = 1 [(gogoproto.nullable) = false];
```

Comment:
doesn't this line need to go away?

Reply:
No? I think we want to keep it so if a pre-refactor node sends a spec, which sets this field, a post-refactor node is able to unpack/repack into the nested version.